spec/airport.spec.js

```diff
@@ -0,0 +1,603 @@
+import Airport from '../src/airport.js';
+import { assertEquals, assertFalse, assertThrows, assertTrue } from '../test_framework.js';
+
+console.log('------------------------------');
+console.log('Running tests for airport.js');
+let passCount = 0, failCount = 0;
+
```

Comment:
Using frameworks like Jasmine would allow for using features like beforeEach, which would reduce redundant test code.